utils/user-product-helper.ts

```diff
@@ -1,12 +1,15 @@
 import { ProductCode } from '@/enums'
 import { ProductI } from '@/interfaces'
 
-export const getProductInfo = (config, type: ProductCode): ProductI => {
+/**
+ * Returns product info object for specified type.
+ */
+export function getProductInfo (config, type: ProductCode): ProductI {
   switch (type) {
     case ProductCode.BUSINESS:
       return {
         image: 'BCRS_dashboard_thumbnail_image.jpg',
-        link: config?.authURL + 'home/maintain-business' || 'link_not_configured',
```

Comment:
First, the regular Auth URL takes you to this page anyway.

Second, we want to avoid hard-coded routes in THIS application.